docs/configuration.md

```diff
@@ -698,39 +698,6 @@ This storage backend uses Amazon's Simple Storage Service (S3).
   </tr>
 </table>
 
-### Maintenance
```

Comment:
Why is this getting moved? This isn't a part of a particular storage driver. This should be in the top of the storage section (not below s3, as it is now).